lib/operation.js

```diff
@@ -474,8 +474,10 @@ function negate (options) {
  * @param {Boolean} [normalise=true]
  * @returns {Sharp}
  */
-function normalise (normalise) {
+function normalise (normalise, lowerBin, upperBin) {
   this.options.normalise = is.bool(normalise) ? normalise : true;
+  this.options.normaliseLowerBin = is.number(lowerBin) ? lowerBin : 1;
```

Comment:
Could validate these via `is.inRange()`.

Reply:
[resolved](https://github.com/lovell/sharp/pull/3578/commits/2b16433d97e4cb95a07369b5e8fa68cc80d9b905)